app/views/businessmatching/updateservice/add/business_applied_for_psr_number.scala.html

```diff
@@ -30,43 +30,52 @@
     @heading("businessmatching.psr.number.title", "summary.updateservice")
 }
 
+@supportingContent = {
+    <p>@m("businessmatching.updateservice.psr.number.hint")</p>
+}
+
 @main(
     title = Messages("businessmatching.psr.number.title") + " - " + Messages("summary.updateservice"),
     heading = header
 )  {
 
     @form(f, controllers.businessmatching.updateservice.add.routes.BusinessAppliedForPSRNumberController.post(edit)) {
-        <p>
-            @m("businessmatching.updateservice.psr.number.hint")
-        </p>
-        @fieldset(
-            f = f("appliedfor-fieldset"),
+
+        @fieldsetWithErrorGroup(
+            f = f("appliedFor"),
             legend = "businessmatching.updateservice.psr.number.title",
-            legendHidden = true,
+            section = "summary.updateservice",
             panel = false,
             classes = Seq("inline")
         ) {
-
-            @errorGroup(
-                field = f("appliedFor")
+            @radio(
+                f = f("appliedFor"),
+                labelText = "lbl.yes",
+                value = "true",
+                target = s"#$target",
+                inline = true
+            )
+            @panel(
+                field = Some(f(s"$target")),
+                id = target
             ) {
-
-                @radio(
-                    f = f("appliedFor"),
-                    labelText = "lbl.yes",
-                    value = "true",
-                    target = s"#$target",
-                    inline = true
-                )
-                @radio(
-                    f = f("appliedFor"),
-                    labelText = "lbl.no",
-                    value = "false",
-                    inline = true
+                @textarea(
```

Comment:
Is this right, input below has not been removed? And probably should be input, not textarea